app/uk/gov/hmrc/gform/sharedmodel/formtemplate/AuthConfig.scala

```diff
@@ -134,20 +137,32 @@ object AuthConfig {
   def toEnrolmentPostCheck(maybeRegimeId: Option[RegimeId]): EnrolmentPostCheck =
     maybeRegimeId.fold(NoCheck: EnrolmentPostCheck)(RegimeIdCheck.apply)
 
+  def enrolmentActionMatch(enrolmentAction: Option[EnrolmentAction]): EnrolmentAction = enrolmentAction match {
```

Comment:
The same as on frontend: 

Body of this method may be shortened to just `enrolmentAction.getOrElse(NoAction)`